src/systemTest/java/com/flightstats/hub/callback/model/RequestObject.java

```diff
@@ -0,0 +1,13 @@
+package com.flightstats.hub.callback.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class RequestObject {
```

Comment:
If this is a model that the hub exposes as part of the webhook callback rather than it being an internal part of the callback server we have, I'd move it up to hub/model instead of hub/callback/model.